core/dataflow/src/main/resources/udf_operator.py

```diff
@@ -0,0 +1,112 @@
+from abc import ABC
```

Comment:
I suggest we create a new folder under `resources`, let's call it `resources/python_udf` and put all related python files there.

Reply:
OK.

Reply:
Done.